frontend/src/components/Login/LoginDesktop.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { styled } from '@mui/material/styles';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import './login.css';
+
+const Item = styled(Paper)(({ theme }) => ({
+    backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  }));
+
+export default function LoginDesktop() {
+    return (
+        <div id="wrapper1">
```

Comment:
this looks great!! i noticed you did find the paper component, but maybe just couldnt figure it out? do you want to pair up sometime this weekend to take a look at getting this to just use the components from the ui framework? putting two heads together is often rlly helpful. i think using the framework is going to be tough at first but once we get used to it, it will speed up development a lot.

Reply:
I played around with some of the grids/boxes that mui has but I didn't have the control I wanted over them, or I just didn't know how to use it to get what I wanted. Mui will definitely be handy in some of the later things, but this page is very simple so I didn't mind using styling.